### PR TITLE
Added Peter Selinger's course.

### PR DESCRIPTION
**Pull request by selinger**
Added my course. I hope I got the markup right.

### CODE REVIEW

File: doc/user-manual/getting-started/tutorial-list.rst [hunk 1]
```diff
@@ -111,7 +111,10 @@ Courses using Agda
 - `Correct-by-construction Programming in Agda
   <https://github.com/jespercockx/ohrid19-agda>`__, a course at the
   EUTYPES Summer School '19 in Ohrid.
-
+- `Lectures on Agda <https://www.mathstat.dal.ca/~selinger/agda-lectures/>`__
+  , a course by Peter Selinger at Dalhousie University, Winter 2021.
+
```
Comment (turion) [inline doc/user-manual/getting-started/tutorial-list.rst]:
```suggestion

```

File: doc/user-manual/getting-started/tutorial-list.rst [hunk 2]
```diff
@@ -111,7 +111,10 @@ Courses using Agda
 - `Correct-by-construction Programming in Agda
   <https://github.com/jespercockx/ohrid19-agda>`__, a course at the
   EUTYPES Summer School '19 in Ohrid.
-
+- `Lectures on Agda <https://www.mathstat.dal.ca/~selinger/agda-lectures/>`__
+  , a course by Peter Selinger at Dalhousie University, Winter 2021.
+  
+
```
Comment (turion) [inline doc/user-manual/getting-started/tutorial-list.rst]: Maybe this extra line needs to be removed as well, I'm not sure.

**Comment by selinger** [comment]
OK, I removed the extra line, and also the two spaces at the beginning of the empty line. Maybe this will work.

**Comment by turion** [comment]
I think it's probably good style to squash the commits. But github can do that as well.

**Comment by andreasabel** [comment]
Thanks, @selinger and @turion!

**Pull request merged**